tests/API/Integration/UnitsTests.cs

```diff
@@ -76,7 +76,6 @@ public async Task GetParksAndRecUnit()
                 Assert.AreEqual(expected.Parent?.Id, actual.Parent?.Id);
             }
 
-            [TestCase(ValidRswansonJwt, TestEntities.Units.ParksAndRecUnitId, PermsGroups.GetPut, Description="As Ron I can a unit I manage")]
```

Comment:
Let's add this test back. What are the new expectations for Ron's permissions?

Reply:
It's back in 67348c2, though `PermGroups.All` might not - ultimately - be the correct permissions.

Reply:
Ron has all the unit permissions, we already have a test case for that, so we've decided to remove this test.